frontend/src/components/Edu.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div>
+    <input
+      type="text"
+      v-model="education.degree"
+      class="degree"
+      placeholder="Degree"
+      @input="saveEducation"
+    />
+
+    <input
+      type="text"
+      v-model="education.university"
+      placeholder="University"
+      @input="saveEducation"
+    />
+  </div>
+
+  <div>
+    <label>From</label>
+    <input
+      type="date"
+      v-model="education.startingYear"
+      @input="saveEducation"
+    />
+
+    <label>></label>
+    <input
+      type="date"
+      v-model="education.graduationYear"
+      @input="saveEducation"
+    />
+  </div>
+
+  <div>
+    <button @click="removeEducation">Remove</button>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    education: Object,
+    index: Number,
+  },
+  methods: {
+    saveEducation() {
+      this.$emit("save", this.index, this.edu);
+    },
+    removeEducation() {
+      this.$emit("remove", this.index);
+    },
+  },
+};
+</script>
+
+<style scoped>
+button {
+  background-color: #f5f5f5;
+
```

Comment:
```suggestion
```

Reply:
also, just me picking out)) but this newline does not add to the code readability